pages/en/billing.mdx

```diff
@@ -2,75 +2,74 @@
+2. Click on the "Connect Wallet" button on the top right corner of the page. Select your wallet and click on "Connect". If you're using [Gnosis-Safe](), you'll be able to connect your multisig as well as your signing wallet. Then, sign the associated transaction. This will not cost any gas.
 
-Lastly, learn how to prioritize price, speed, data freshness (among other factors) for your queries by watching [this video](https://www.loom.com/share/b5fc533e48584cb694017392c80c75e0). If you have any questions, please ask in [The Graph Discord](https://discord.gg/fjDWCTZE) and the community will be happy to help.
+3. Click the 'Add GRT' button at the center of the page. A side panel will appear.
 
-### How to Fill Your Account With a Multisig Wallet
+4. Enter the amount of GRT you would like to use and click the button to allow access to your GRT. This transaction will not require a gas fee. Ensure all signers have approved the transaction, and execute the transaction.
 
-Multisigs are smart contracts that can exist only on the network they have been created, so if you created one on Ethereum Mainnet - it will only exist on Mainnet. Since our billing uses Polygon, if you were to bridge GRT to the multisig address on Polygon the funds would be lost.
+5. Click 'Allow GRT Access' to allow the Subgraph Studio to access your GRT. Sign the associated transaction in your wallet. This will not cost any gas.
 
-To overcome this issue, we created [a dedicated tool](https://multisig-billing.thegraph.com/) that will help you deposit GRT on our billing contract (on behalf of the multisig) with a standard wallet / EOA (an account controlled by a private key).
+6. Specify the amount of GRT you would like to use and click the 'Add GRT to billing balance' button to add GRT to your account. This transaction will require a gas fee. Ensure all signers have approved the transaction and execute it.
 
-You can access our Multisig Billing Tool here: https://multisig-billing.thegraph.com/
+7. Once the transaction is confirmed, you'll see the GRT added to your account within an hour.
 
-This tool will guide you to go through the following steps:
+### Withdrawing GRT using a multisig wallet
 
-1. Connect your standard wallet / EOA (this wallet needs to own some ETH as well as the GRT you want to deposit)
-2. Bridge GRT to Polygon. You will have to wait 7-8 minutes after the transaction is complete for the bridge transfer to be finalized.
-3. Once your GRT is available on your Polygon balance you can deposit them to the billing contract while specifying the multisig address you are funding in the `Multisig Address` field.
+> This section is written assuming you have deposited GRT into your billing account on [Subgraph Studio](https://staging.thegraph.com/studio/billing/) and that you're on the Arbitrum network.
```

Comment:
For multisig, they'll still be on Mainnet

Reply:
Good catch!